app/placeholder/placeholder-deploy.sh

```diff
@@ -17,6 +17,23 @@
 # any errors? exit immediately.
 set -e
 
+# if deploying with a suffix, adjust the config to suit the custom site
+# https://firebase.google.com/docs/hosting/multisites#set_up_deploy_targets
+if [[ -n $SUFFIX ]]; then
+    json -I -f firebase.json -e "this.hosting.target='$SUFFIX'"
+    UPDATED=true
+
+    echo "{\"projects\": {}, \"targets\": {\"${PROJECT_ID}\": {\"hosting\": {\"${SUFFIX}\": [\"${PROJECT_ID}-${SUFFIX}\"]}}},\"etags\": {}}" | json > .firebaserc
+    echo "Customised .firebaserc created to support site."
+    cat .firebaserc
+fi
```

Comment:
future: I've noticed in previous PRs that this is approaching readability limits. Using this moment to note that we may want to consider switching to something more readable.

A couple options include using [`jq` to render JSON](https://www.benjaminrancourt.ca/how-to-easily-create-a-json-file-in-bash/) or maybe using a template system like `envsubst`.

Reply:
Moved to envsubst